### PR TITLE
Enable Metal3 plugin in Console

### PR DESCRIPTION
**Pull request by jtomasek**

### CODE REVIEW

File: frontend/packages/console-app/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@console/dev-console": "0.0.0-fixed",
+    "@console/metal3-plugin": "0.0.0-fixed",
```
Comment (spadgett) [inline frontend/packages/console-app/package.json]: nit: can we keep the list alphabetized?

Comment (jtomasek) [reply frontend/packages/console-app/package.json]: Done

**Pull request merged**